### PR TITLE
Fix the login window display issue

### PR DESCRIPTION
**Pull request by zeineb**

### CODE REVIEW

File: common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java [hunk 1]
```diff
@@ -437,13 +445,17 @@ public void onClick(ClickEvent event) {
         String cacheLogin = Settings.get().getSetting(controller.getLoginSettingKey());
         // check if username cookie variable is set
         // if not, the login input text is set to the cacheLogin
-        if (!getCookieUserName().isEmpty() && getCookieUserName().toLowerCase().trim() != "null") {
-            form.setValue("login", getCookieUserName());
+        if (getCookieUserName() == null) {
+            form.setValue("login", cacheLogin);
```
Comment (marcocast) [inline common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java]: should we check if (cacheLogin != null)  like we do below?

File: common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java [hunk 2]
```diff
@@ -311,6 +313,12 @@ public void onChanged(ChangedEvent event) {
         this.layout.draw();
     }
 
+    // console log in GWT
+    // wrapper to console.log following this: https://stackoverflow.com/questions/9247424/how-to-print-to-the-console-in-gwt 
+    native void consoleLog(String message) /*-{
+                                           console.log(message);
+                                           }-*/;
+
```
Comment (chloegugli) [inline common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java]: Do you need this?

File: common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java [hunk 3]
```diff
@@ -437,13 +445,17 @@ public void onClick(ClickEvent event) {
...
         } else {
-            if (cacheLogin != null) {
-                form.setValue("login", cacheLogin);
+            if (!getCookieUserName().isEmpty() && getCookieUserName().toLowerCase().trim() != "null") {
+                form.setValue("login", getCookieUserName());
             } else {
-                form.setValue("login", "");
+                if (cacheLogin != null) {
+                    form.setValue("login", cacheLogin);
+                } else {
```
Comment (Mykhailenko) [inline common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java]: form.setValue("login", cacheLogin != null ? cacheLogin : "")

File: common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java (hunk 2 above)
Comment (zeineb) [reply common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java]: I added it to log (debug), as in GWT it is not possible to do a proper console.log, @amouhoub faced this same issue.
I will remove it.

File: common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java (hunk 3 above)
Comment (zeineb) [reply common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java]: Thanks!

**Pull request merged**